packages/algorand-js/src/interpreter/opcode-list.ts

```diff
@@ -199,8 +232,19 @@ export class Intcblock extends Op {
   readonly intcblock: Array<bigint>;
   readonly interpreter: Interpreter;
 
-  constructor (interpreter: Interpreter, intcblock: Array<bigint>) {
+  /**
+   * @param args words list extracted from line
+   * @param line line number in TEAL file
+   * @param interpreter interpreter object
+   */
+  constructor (args: string[], line: number, interpreter: Interpreter) {
     super();
+    const intcblock: Array<bigint> = [];
```

Comment:
```suggestion
    const intcblock: BigInt[];
```

Reply:
this dosn't work. because this should initialized before use